plugins.json

```diff
@@ -9,4 +9,5 @@
     "ghishadow/lapce-catppuccin",
     "felipetesc/lapce-sweetraw-theme",
     "superlou/lapce-python"
```

Comment:
You need a comma at line end here for a valid json

Reply:
Thank you, I didn't notice that